server/pytest_devpi_server/__init__.py

```diff
@@ -16,5 +16,5 @@ def devpiserver_makepypistage():
     def makepypistage(xom):
         from devpi_server.main import _pypi_ixconfig_default
         return PyPIStage(xom, username="root", index="pypi",
-                         ixconfig=_pypi_ixconfig_default)
+                         ixconfig=dict(_pypi_ixconfig_default))
```

Comment:
comment why doing a copy of dict is useful

Reply:
Add in 0a59c7f6